calico/getting-started/kubernetes/self-managed-onprem/onpremises.mdx

```diff
@@ -59,6 +59,12 @@ installed directly on the cluster as a Deployment, and is configured through one
    ```
    kubectl create -f custom-resources.yaml
    ```
+1. Verify {{prodname}} installation in your cluster.
+
+   ```
+   watch kubectl get pods -n calico-system
+   ```
```

Comment:
IMO, this is only verification if you know what the expected output is. Perhaps include an example output from a known-good system?

Reply:
Yes! 

We should have a code block with the title: Example output. And it would be nice to get more information. Something like, If this worked, then you should see this ...